scripts/us_census/pep/us_pep_sexrace/postprocess.py

```diff
@@ -0,0 +1,374 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to read all
+the generated csv for
+National, State and County
+and generate final csv, MCF, TMCF file
+"""
+
+import pandas as pd
+import os
+import enum
+
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+
+
+class Outputfiles(enum.Enum):
+    NationalBefore2000 = 1
+    StateCountyBefore2000 = 2
+    StateCountyAfter2000 = 3
+    NationalAfter2000 = 4
+
+
+def write_to_tmcf(filename: str, tmcf: str):
+    with open(_CODEDIR + "/output_files/final/" + filename + ".tmcf",
+              'w+',
+              encoding='utf-8') as f_out:
+        f_out.write(tmcf.rstrip('\n'))
+
+
+def write_to_mcf(filename: str, final_mcf_template: str):
+    with open(_CODEDIR + "/output_files/final/" + filename + ".mcf",
+              'w+',
+              encoding='utf-8') as f_out:
+        f_out.write(final_mcf_template.rstrip('\n'))
+
+
+def process_national_before_2000(df1: pd.DataFrame):
+    # coverting year values to int
+    df1['Year'] = df1['Year'].astype(float).astype(int)
```

Comment:
Since this is within a function, you could drop the "1" from "df1"?

Same comment for "df3", "df5", etc. in the functions below.

Reply:
Dropped 1,3 and 5 from withnin the function.